### PR TITLE
[FEATURE] Add "configPath" option

### PR DESCRIPTION
**Pull request by matz3**
Fixes: https://github.com/SAP/karma-ui5/issues/169

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
   - [url](#url)
   - [type](#type)
   - [paths](#paths)
+  - [configPath](#configPath)
```
Comment (petermuessig) [inline README.md:16]: Is it a path or a file?

Comment (matz3) [reply README.md:16]: Path to the file 😄
The CLI has `--config`, but here config is already used to provide the UI5 bootstrap config...
So I decided to use the API option name, which is `configPath`. See https://sap.github.io/ui5-tooling/api/module-@ui5_project.normalizer.html

Comment (petermuessig) [reply README.md:16]: OK if already used in the UI5 Tooling already, I am fine with it

File: lib/framework.js [hunk 2]
```diff
@@ -462,7 +466,10 @@ class Framework {
 			config.beforeMiddleware.push("ui5--pauseRequests");
 			config.middleware.push("ui5--serveResources");
 			// config.middleware.push("ui5--serveThemes");
-			server = await this.setupUI5Server(config.basePath);
+			server = await this.setupUI5Server({
+				basePath: config.basePath,
+				configPath: config.ui5.configPath
```
Comment (petermuessig) [inline lib/framework.js:471]: could `ui5` be `undefined` if not used in the configuration => causing an undefined exception?

Comment (codeworrior) [reply lib/framework.js:471]: Then it would have failed in line 462 already...

Comment (petermuessig) [reply lib/framework.js:471]: OK - wasn't visible in the review

Comment (petermuessig) [reply lib/framework.js:471]: even line 465 wasn't visible to me in the review... 😄

**Pull request merged**